application/testing/CMakeLists.txt

```diff
@@ -150,6 +150,7 @@ f3d_test(NAME TestVTM DATA mb.vtm UI)
 f3d_test(NAME TestVTK DATA cow.vtk)
 f3d_test(NAME TestNRRD DATA beach.nrrd ARGS -s)
 f3d_test(NAME TestSPLAT DATA small.splat ARGS -osy --up=-Y --point-sprites-size=1)
+f3d_test(NAME TestQuakeMDL DATA glaunch.mdl ARGS --animation-index=1 --animation-autoplay=true --animation-time=0.1)
```

Comment:
You also need to add DefaultConfig and ThumbnailConfig tests (see other tests named like this)